server/livecheck.py

```diff
@@ -60,10 +95,19 @@ def processCodeData(data):
     # verify certificate
     root_cert_file = makeTempFile(root_cert_text)
     cert_file = makeTempFile(certificate)
+    cert_details = parseCertDetails(cert_file)
 
     cert_verification_result = call(['openssl', 'verify', '-CAfile', root_cert_file, cert_file])
+    is_dev = False
     if cert_verification_result != 0:
-        return None
+        # attempt to verify with dev certificate as a fallback
+        call(['rm', root_cert_file])
+        root_cert_file = makeTempFile(root_cert_text_dev)
+        cert_verification_result_dev = call(['openssl', 'verify', '-CAfile', root_cert_file, cert_file])
+        if cert_verification_result_dev != 0:
+            call(['rm', f"{root_cert_file} {cert_file}"])
```

Comment:
I think previously if `cert_verification_result != 0`, we didn't clean up the temp files creates on lines 96-97

Reply:
Good catch!

Reply:
good catch indeed.